core/src/main/kotlin/br/com/concrete/yosef/api/DynamicViewCreator.kt

```diff
@@ -57,14 +60,45 @@ class DynamicViewCreator(
                     "not a ViewGroup.")
             }
 
-            it.forEach { addChildrenRecursively(parentView, it, listener) }
+            if (parentView is ListView) {
```

Comment:
this is a no for me, we need to find a way of not exposing implementation details here  (we can consider breaking the api, but I think I can come up with something that won't require this)